quickstarts/microsoft.digitaltwins/digitaltwins-with-function-time-series-database-connection/azuredeploy.parameters.json

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "digitalTwinsName": {
+            "value": "GEN-UNIQUE"
+        },
+        "eventHubsNamespaceName": {
+            "value": "GEN-UNIQUE"
+        },
+        "eventHubName": {
+            "value": "GEN-UNIQUE"
+        },
+        "adxClusterName": {
+            "value": "GEN-UNIQUE"
+        },
+        "databaseName": {
+            "value": "GEN-UNIQUE"
+        },
+        "databaseTableName": {
+            "value": "GEN-UNIQUE"
+        },
+        "eventHubsNamespacePlan": {
+            "value": "Basic"
+        },
+        "eventHubsNamespaceTier": {
+            "value": "Basic"
+        },
+        "eventHubsNamespaceCapacity": {
+            "value": 1
+        },
+        "clusterPlan": {
+            "value": "Dev(No SLA)_Standard_D11_v2"
+        },
+        "clusterCapacity": {
+            "value": 1
+        },
+        "clusterTier": {
+            "value": "Basic"
+        },
+        "retentionInDays": {
+            "value": 1
+        },
+        "partitionCount": {
+            "value": 2
+        },
+        "hotCachePeriod": {
+            "value": "P30D"
+        },
+        "softDeletePeriod": {
+            "value": "P1Y"
+        },
+        "principalId": {
+            "value": "GEN-AZUREAD-OBJECTID"
+        },
+        "principalType": {
+            "value": "ServicePrincipal"
+        },
+        "location": {
+            "value": "westus2"
```

Comment:
is this globally available?  if so, remove this so the defaultValue can be tested, if not add allowedValues to main.bicep and remove the defaultValue.

Reply:
The digital twins cannot be created in any region. I added an allowedValues list as requested. However, I did not remove the defaultValue here since problems could occur if [resourceGroup().location] or [deployment().location] is used as a default value in the main.bicep if the location is not in the allowed list.